Updated Changi TestNet Guides (#17)
DeFiCh/handbook: src/guides/guide_changi.md

>
> `RUST_LOG=debug defid -changi --daemon`

> Once the node is running it should start syncing. The process of syncing can a couple of hours depending on your machine so just be patient.

Reviewer: ```suggestion
Once the node is running it should start syncing. The process of syncing can take a couple of hours depending on your machine.
```

### Connect Metamask.

To connect to MetaMask the process is the same as the one described in [Connect Metamask to Changi TestNet](./guide_changi_short.md) you can refer to that guide for more information on it. 

> > Do note that the RPC url will need to be pointed to your local node and not http://changi.dfi.team

Reviewer: ```suggestion
> Do note that the RPC url will need to be pointed to your local node (Default: `http://127.0.0.1:20551`) and not http://changi.dfi.team
```
